sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -214,7 +215,7 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
     this._context = context;
   }
 
-  private async _init(abortSignal?: AbortSignalLike): Promise<void> {
+  private async _init(operationOptions: OperationOptionsBase = {}): Promise<void> {
```

Comment:
It's possible this is too big for this PR, but is it possible this can be made non-optional as well? I'd like the caller to explicitly opt out of passing an abortSignal since this is such a low-level piece of code.

Reply:
Yes